Assets/Scripts/Framework/Gameplay/LaneSystem/LaneManager.cs

```diff
@@ -45,15 +45,13 @@ private void Start()
 
         private void RemoveNoteFromLane(HitStatus hitStatus, int laneID)
         {
-            if(hitStatus != HitStatus.Miss)
-            {
-                Note target = _lanes[laneID].Notes[0];
-                _lanes[laneID].Notes.Remove(target);
+            if (hitStatus == HitStatus.Miss)
```

Comment:
Do we also need to check for HitStatus.MissClick?

Reply:
No, when it's a MissClick it means there is no note in range to be hit and we don't want to do anything besides deduct points and health. Which is done by a different system.